Add deprecation notice, release 0.0.4

patrickrobrecht opened the pull request:
Add notice that Checksum Verifier is deprecated. Fixes #18.

AntiVirus 1.4.0 is released to wp.org SVN now.

After the merge this repository can be archived.

patrickrobrecht committed:
Add deprecation notice

patrickrobrecht committed:
Cleanup

patrickrobrecht committed:
Add deprecation notice and hints how to switch to AntiVirus

patrickrobrecht committed:
Release final version 0.0.4

krafit commented on inc/class-checksum-verifier.php:
```diff
@@ -305,4 +301,27 @@ public static function uninstall_hook() {
+			<p><?php esc_html_e( 'Checksum Verifier is deprecated. Its functionality is integrated into AntiVirus now.', 'checksum-verifier' ); ?></p>
+			<p><?php esc_html_e( 'Switch to AntiVirus now:', 'checksum-verifier ' ); ?></p>
+			<ol>
+				<li><?php esc_html_e( 'Install and activate AntiVirus 1.4.0+.', 'checksum-verifier' ); ?></li>
```
I'd suggest, we add a link in this step, referencing `/wp-admin/plugin-install.php?s=pluginkollektiv&tab=search&type=author` so the user gets the right plugins.

stklcode replied:
Just wanted to type the same suggestion.
The settings page link works fine, if AntiVirus is already installed, but obviously points to nothing otherwise. Linkint the plugin setup would be convenient here.

patrickrobrecht replied:
If added the link to the installation page (shown if AntiVirus is not active). The link to the settings page is shown only if AntiVirus is active - this was intended before, but there was a bug in my code which is fixed now.

patrickrobrecht committed:
Add link for AntiVirus installation, fix settings link to be shown if AntiVirus is activated

stklcode reviewed (approved):
We could remove the _package.json_ and _Gulpfile.json_ or at least correct/remove the version number 0.0.2 to clean up the repo as both are not actively used anyway.

Apart from that the changes work for me.

patrickrobrecht committed:
Remove unused package.json and Gulpfile.js, add export-ignore for vendor

patrickrobrecht commented:
> We could remove the _package.json_ and _Gulpfile.json_ or at least correct/remove the version number 0.0.2 to clean up the repo as both are not actively used anyway.

As I touched the other setup files already, I've removed them as well. In addition I've added `composer.lock` and `vendor` to export-ignore in `.gitattributes`.

stklcode commented:
Sounds reasonable.
